oaknational/oak-ai-lesson-assistant
# fix(share): fix share bug

## Description

- adds `isShared` to Aila package, so that it persists, fixing the share 404 bug
- adds authorisation step in `chatHandler()`

## Issue(s)

https://www.notion.so/oaknationalacademy/Share-Page-404-dd392621286e40279204d563960f2e75?pvs=4

## How to test

1. Go to the site
2. Create a lesson
3. Share it
4. Go back to lesson and change it
5. Check the share page ... should still be shared!
